roadmap.md

```diff
@@ -25,22 +24,24 @@ Architecture:
 
```

Comment:
It would be good to double check links. This one for example is broken: https://github.com/dotnet/cli/blob/master/Documentation/intro-to-cli.md